setup.py

```diff
@@ -263,18 +263,20 @@ def _pkg_config(name):
             if not DEBUG:
                 command_libs.append("--silence-errors")
                 command_cflags.append("--silence-errors")
-            libs = (
+            libs = re.split(
+                r"\s*-L",
                 subprocess.check_output(command_libs, stderr=stderr)
                 .decode("utf8")
-                .strip()
-                .replace("-L", "")
+                .strip(),
             )
-            cflags = (
-                subprocess.check_output(command_cflags)
+            libs.remove("")
+            cflags = re.split(
+                r"\s*-I",
+                subprocess.check_output(command_cflags, stderr=stderr)
```

Comment:
Did you want to briefly explain why you added `, stderr=stderr` here? It was added to the libs command in #6261 to avoid a `Unknown option --keep-system-libs` error, but that doesn't apply to the cflags command.

Reply:
To be consistent with the other check_output call, since only STDOUT is used to get the paths.